pom.xml

```diff
@@ -4,25 +4,28 @@
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 
-	<groupId>org.springframework</groupId>
-	<artifactId>sgh-email-adapter</artifactId>
+	<groupId>uk.nhs.digital</groupId>
+	<artifactId>email-adapter</artifactId>
 	<version>0.1.0</version>
 	<packaging>jar</packaging>
-	<name>Spring Boot Docker</name>
-	<description>Getting started with Spring Boot and Docker</description>
+	<name>IUCDS pilot</name>
+	<description>IUCDS pilot email adapter</description>
 
 	<parent>
-		<groupId>org.springframework.boot</groupId>
+		<groupId>com.cooldatasoft</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.2.1.RELEASE</version>
-		<relativePath />
+		<version>2.2.5.RELEASE</version>
 	</parent>
 
 	<properties>
-		<docker.image.prefix>iucds</docker.image.prefix>
+		<docker.image.prefix>nhsd</docker.image.prefix>
 		<java.version>11</java.version>
 		<aws-sdk.version>1.11.362</aws-sdk.version>
 		<dockerfile-maven-plugin.version>1.4.13</dockerfile-maven-plugin.version>
```

Comment:
version are defined in the parent pom already. Unless you are overriding with a different version there is no need to define the plugin version properties here.